packages/firebase_app_check/firebase_app_check/android/build.gradle

```diff
@@ -55,10 +55,9 @@ android {
   dependencies {
     api firebaseCoreProject
     implementation platform("com.google.firebase:firebase-bom:${getRootProjectExtOrCoreProperty("FirebaseSDKVersion", firebaseCoreProject)}")
-    // TODO: Since App Check is a beta SDK it's not available in the Firebase Android BoM so we need to specify an exact version here.
-    implementation 'com.google.firebase:firebase-appcheck-safetynet:16.1.0'
-    implementation 'com.google.firebase:firebase-appcheck-debug:16.1.0'
+    implementation 'com.google.firebase:firebase-appcheck-debug'
     implementation 'com.google.firebase:firebase-appcheck-playintegrity'
+    implementation 'com.google.firebase:firebase-appcheck-safetynet'
```

Comment:
Moved it to the bottom as it is deprecated and, potentially, is a subject for removal in future: https://developer.android.com/training/safetynet/deprecation-timeline